app/components/global/footer.tsx

```diff
@@ -57,15 +57,16 @@ const MobileFooter = () => {
 							<span className="text-xs">More</span>
 						</div>
 					</DrawerTrigger>
-					<DrawerContent className=" gap-10">
+					<DrawerContent className="gap-10">
 						<DrawerHeader>
 							<div className="flex justify-center">
 								<img className="h-10 w-10" src={logo} alt="VoiceDeck Logo" />
 							</div>
 						</DrawerHeader>
 						{externalLinks.map((link) => (
-							<a
-								href={link.url}
+							<Link
```

Comment:
I don't think you want to use the Link component for external links unless that has recently changed. Link assumes all links are internal.

We can write a component that determines whether it is internal or external though.

https://github.com/remix-run/remix/discussions/4888

Reply:
you're right. didn't register as external to me, moving to fast!

Reply:
Reverted now!